dotnetv3/Glacier/README.md

```diff
@@ -20,11 +19,11 @@ Code excerpts that show you how to call individual service functions.
 * [AddTagsToVaultExample](AddTagsToVaultExample/) - Add tags to an Amazon S3 Glacier vault.
 * [CreateVaultExample](CreateVaultExample/) - Create an Amazon S3 Glacier vault.
 * [DescribeVaultExample](DescribeVaultExample/) - Describe an Amazon S3 Glacier vault.
-* [DownloadArchiveHighLevelExample](DownloadArchiveHighLevelExample/) - Download an Amazon S3 Glacier archive by using the high level API.
+* [DownloadArchiveHighLevelExample](DownloadArchiveHighLevelExample/) - Download an Amazon S3 Glacier archive by using the high-level API.
 * [ListJobsExample](ListJobsExample/) - List information about current Amazon S3 Glacier jobs.
 * [ListTagsForVaultExample](ListTagsForVaultExample/) - List tags for an Amazon S3 Glacier vault.
 * [ListVaultsExample](ListVaultsExample/) - List the Amazon S3 Glacier vaults for an AWS Region.
-* [UploadArchiveHighLevelExample](UploadArchiveHighLevelExample/) - Upload an Amazon S3 Glacier archive by using the high level API.
+* [UploadArchiveHighLevelExample](UploadArchiveHighLevelExample/) - Upload an Amazon S3 Glacier archive by using the high-level API.
```

Comment:
The format of these links should be:

```
[title_abbrev](relative URL to example code file) (ServiceAction)
```

So for create vault, this would be `[Create a vault](CreateVaultExample/CreateVault.cs) (CreateVault)`

* `title_abbrev` comes from [glacier_metadata.yaml](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.doc_gen/metadata/glacier_metadata.yaml).

The reason for using the exact title_abbrev text is that when customers click through from the docs, they'll find the same title they were just looking at and so more easily find the code they need.

The reason to list the service action is to help when customers search for an action by name.

Reply:
Updated.